docs/user-guide/installation.rst

```diff
@@ -141,17 +141,29 @@ Tools like `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ and `virtualenv
 
 .. tabs::
 
-    .. tab:: Rasa and Rasa X
+    .. tab:: Rasa Open Source and Rasa X
 
-        To install both Rasa and Rasa X in one go:
+        First make sure your ``pip`` version is up to date:
+
+        .. code-block:: bash
+
+            $ pip install -U pip
```

Comment:
Why are we updating `pip` now?

Reply:
from description:
- add info about upgrading pip (to avoid #4784)`
https://forum.rasa.com/t/error-collecting-tensorflow-1-15-0/21306